eng/Versions.props

```diff
@@ -335,6 +335,7 @@
     <MicrosoftDataSqlClientVersion>4.0.5</MicrosoftDataSqlClientVersion>
     <MicrosoftAspNetCoreAppVersion>6.0.0-preview.3.21167.1</MicrosoftAspNetCoreAppVersion>
     <MicrosoftOpenApiVersion>1.6.13</MicrosoftOpenApiVersion>
+    <MicrosoftOpenApiReadersVersion>1.6.13</MicrosoftOpenApiReadersVersion>
```

Comment:
Can these get out of sync?

Reply:
If we're not careful yes, although there should be no issue as long as they target the same major version.

Our package reference resolver uses `{package name sans periods } + "Version"` as the property name used to do version looks so we unfortunately have to use two different properties here. :/